docs/how-to/service-placement.rst

```diff
@@ -0,0 +1,71 @@
+Start Ceph services on any node of the MicroCeph cluster
+========================================================
+
+MicroCeph allows users to start supported ceph service on any target host in the MicroCeph cluster. This can be achieved by providing the remote hostname as the --target parameter to the enable command (as described below)
```

Comment:
I wonder if we should lead in with a sentence about motivation and context, something like:

> MicroCeph will automatically start Mon, Mgr, and Mds services on the first three deployed nodes in the cluster. Users who wish to enable additional instances of these services for scalability or resilience can do so by utilizing the `enable` command and specifying a node via the `--target` parameter.